websites/R/RTLplay/presence.ts

```diff
@@ -0,0 +1,555 @@
+						)[0].textContent;
+
+						presenceData.largeImageKey = getChannel(
+							pathname.split("/")[3]
+						).logo;
+						presenceData.largeImageText = getChannel(
+							pathname.split("/")[3]
+						).channel;
+
+						if (time) {
+							presenceData.endTimestamp = presence.getTimestamps(
+								presence.timestampFromFormat(
+									document
+										.querySelector("span.playerui__controls__stat__time")
+										.textContent.split("/")[0]
+										.trim()
+								),
+								presence.timestampFromFormat(
+									document
+										.querySelector("span.playerui__controls__stat__time")
+										.textContent.split("/")[1]
+										.trim()
+								)
+							)[1];
+						}
+
+						if (buttons) {
+							presenceData.buttons = [
+								{
+									label: (await strings).buttonWatchStream,
+									url: href, // We are not redirecting directly to the raw video stream, it's only the media page
+								},
+							];
+						}
+					}
+					break;
+				}
+				case hostname === "www.radiocontact.be": {
+					presenceData.name = getChannel("contact").channel;
+					presenceData.type = getChannel("contact").type;
+
+					if (exist('button[aria-label="stop"]')) {
+						presenceData.smallImageKey = Assets.Listening;
+						presenceData.smallImageText = strings.listeningMusic;
+					} else {
+						presenceData.smallImageKey = Assets.Stop;
+						presenceData.smallImageText = strings.pause;
+					}
+
+					if (!privacy) {
+						// Fetch the data from the API
+						const response = await fetch(
+								"https://core-search.radioplayer.cloud/056/qp/v4/events/?rpId=1"
+							), // Website backend use radioplayer api
+							dataString = await response.text(),
+							data = JSON.parse(dataString);
+
+						if (data.results.now.type === "PE_E") {
+							// When a song is played
+							presenceData.details = data.results.now.name;
+							presenceData.state = data.results.now.artistName;
+							presenceData.largeImageKey = await getShortURL(
+								data.results.now.songArtURL
+							);
+						} else {
+							// When there's no song and only the show
+							presenceData.details = data.results.pis[0].programmeName;
+							presenceData.state = data.results.pis[0].programmeDescription;
+							presenceData.largeImageKey = await getShortURL(
+								data.results.pis[0].imageUrl
+							);
+						}
+
+						presenceData.largeImageText = getChannel("contact").channel;
+
+						if (buttons) {
+							presenceData.buttons = [
+								{
+									label: (await strings).buttonListenAlong,
+									url: href, // We are not redirecting directly to the raw video stream, it's only the media page
+								},
+							];
+						}
+					}
+					break;
+				}
+			}
+			break;
+		}
+
+		/* MEDIA PLAYER PAGE (Lecteur video)
+
+		(https://www.rtlplay.be/rtlplay/player/75e9a91b-29d1-4856-be8c-0b3532862404) */
+		case pathname.split("/")[2] === "player": {
+			const {
+				mediaName = document.querySelectorAll("h1.lfvp-player__title")[0]
```

Comment:
If you're getting the first element user querySelector() instead of querySelectorAll()
```suggestion
				mediaName = document.querySelector("h1.lfvp-player__title")
```